tests/Main.hs

```diff
@@ -212,12 +212,20 @@ main = Hspec.hspec $ do
         predeterminedDateTime
     it "tests whether the fixed packed register has a valid attestation" $
       registerTestFromFile
-        "tests/responses/attestation/packed-02.json"
+        "tests/responses/attestation/packed-03.json"
```

Comment:
@ErinvanderVeen Why is this changed? :thinking:

Reply:
Because we tested 02 twice and 03 never, figured we should fix that :)